client-lib-tests/src/test/java/com/ibm/ws/repository/strategies/test/AddThenHideOldStrategyUnitTest.java

```diff
@@ -57,18 +58,37 @@
 
     private static AddThenHideOldStrategy _athos = new AddThenHideOldStrategy();
 
+    private Object invoke(Object object, String methodName, Object... args) throws SecurityException, IllegalArgumentException, ClassNotFoundException, NoSuchMethodException, IllegalAccessException, InstantiationException, InvocationTargetException {

```

Comment:
Can we/should we use a common method in the ReflectionTricks class (test-utils project)?
Which would save writing this method multiple times

Reply:
Probably :)
In fact its quite similar to the existing reflectiveCall... methods in ReflectionTricks
Only problem is the method is actually slightly different in each class, but it should be doable - I will have a go.